File: mobile/lib/features/visitors/data/daily_helper.dart
class DailyHelper {
  DailyHelper({
    required this.id,
    required this.name,
    required this.phone,
    required this.helperType,
    this.photoUrl,
  });

  factory DailyHelper.fromJson(Map<String, dynamic> json) {
    return DailyHelper(
      id: json['id'] as int,
      name: json['name'] as String,
      phone: json['phone'] as String,
      helperType: json['helper_type'] as String? ?? 'other',
      photoUrl: json['photo_url'] as String?,
    );
  }

  final int id;
  final String name;
  final String phone;
  final String helperType;
  final String? photoUrl;

  static const types = ['maid', 'cook', 'driver', 'nanny', 'gardener', 'other'];
}

File: app/Models/Tenant/DailyHelper.php
<?php

namespace App\Models\Tenant;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;
use Illuminate\Database\Eloquent\SoftDeletes;

class DailyHelper extends Model
{
    use SoftDeletes;

    protected $connection = 'society';

    public const TYPES = ['maid', 'cook', 'driver', 'nanny', 'gardener', 'other'];

    protected $fillable = [
        'flat_id',
        'added_by_user_id',
        'name',
        'phone',
        'helper_type',
        'photo_path',
    ];

    public function flat(): BelongsTo
    {
        return $this->belongsTo(Flat::class);
    }
}
